setup.py

```diff
@@ -262,7 +261,7 @@ def run(self):
     'pytest-rerunfailures',
     'mock',
     'cython',
-    # temporarily remove pyemd to work around appveyor issues
+    # Temporarily remove pyemd to work around appveyor issues. XXX done?
```

Comment:
@mpenkov @menshikh-iv I'd appreciate comments on the three `XXX` in this PR. Why is that code there? Is it still needed?

I'll add your comments directly to the source.

Reply:
If I remember correctly, `pyemd` has broken installation on windows

Reply:
About `nmslib`, ping @masa3141 https://github.com/RaRe-Technologies/gensim/pull/2417/commits/acff16674a5d1bbebfb37149a4d0dfd84333d8cd, can you explain please this one
```python
if (3, 0) < sys.version_info < (3, 7):
    linux_testenv.extend(['nmslib'])
```
why this needed?

Reply:
About `tensorflow/keras`: this was done in https://github.com/RaRe-Technologies/gensim/pull/2211 and needed to avoid some testing issues (not sure how relevant this after 2 years)